### PR TITLE
Bound count arguments throw error

### PR DESCRIPTION
**Pull request by dlangevin**
This is the PR I referenced in #49

- Arguments that come in as Streams to the `rules()` function were throwing an error
- Bound arguments passed into the `t()` helper were not triggering an update of the parent Stream

These are really two related issues, so I can break it into two PRs if you prefer

### CODE REVIEW

File: addon/rules/base.js [hunk 1]
```diff
@@ -3,9 +3,12 @@ import Ember from 'ember';
 export default function(keys, value, result, path, countryCode, fn) {
   var type;
 
+  // Handle results that come in as a Stream
+  if(value.value) { value = value.value(); }
```
Comment (rwjblue) [inline addon/rules/base.js]: Please use the `read` helper.

**Comment by dlangevin** [comment]
@rwjblue updated with `read()`

**Comment by rwjblue** [comment]
:+1:

**Comment by bcardarella** [comment]
Thanks!

**Pull request merged**